docs/getting-started/GETTING_STARTED.md

```diff
@@ -392,3 +392,458 @@ or
 
 In all scenarios, even if the user gives a non-temporary table, only temporary
 tables will actually be deleted.
+
+## Configuring default schema
+
+We can configure the default schema that will be used for all operation involving database.
+environment variable :
+```shell
+AIRFLOW__ASTRO_SDK__SCHEMA="tmp"
+```
+or by updating Airflow's configuration
+```editorconfig
+[astro_sdk]
+schema = "tmp"
+```
+We can fur<h1 align="center">
+  Astro Python SDK
+</h1>
+  <h3 align="center">
+  Tutorial<br><br>
+</h3>
```

Comment:
This should probably live in a dedicated page, within our Sphinx documents, as proposed by @kaxil

Reply:
Make sense, I'll move it.